doc/user/content/sql/tail.md

```diff
@@ -234,7 +234,7 @@ BEGIN;
 DECLARE c CURSOR FOR TAIL t;
 ````
 
-Now use [`FETCH`](/sql/fetch) in a loop to retrieve some number of rows within a time window:
+Now use [`FETCH`](/sql/fetch) in a loop to retrieve each batch of results as soon as they are ready:
 
```

Comment:
It should be "as soon as it is ready" if we are retrieving a batch of bundled results at the same time and "as soon as they are ready" if we retrieve each result one at a time as each occurs (but in that case "batch" is misleading).